libraries/botbuilder-ai/src/luis-client/model/models.ts

```diff
@@ -0,0 +1,215 @@
+/**
+ * @module botbuilder-ai
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+/**
+ * NOTE: This class was auto generated by OpenAPI Generator (https://openapi-generator.tech)
+ * and was manually modified to make it compliant with the current implementation of the library.
+ */
+
+export * from './aPIError';
+export * from './compositeChildModel';
+export * from './compositeEntityModel';
+export * from './entityModel';
+export * from './intentModel';
+export * from './luisResult';
+export * from './sentiment';
+
+import { APIError } from './aPIError';
+import { CompositeChildModel } from './compositeChildModel';
+import { CompositeEntityModel } from './compositeEntityModel';
+import { EntityModel } from './entityModel';
+import { IntentModel } from './intentModel';
+import { LuisResult } from './luisResult';
+import { Sentiment } from './sentiment';
+
+/* tslint:disable:no-unused-variable */
+let primitives = [
+                    'string',
+                    'boolean',
+                    'double',
+                    'integer',
+                    'long',
+                    'float',
+                    'number',
+                    'any'
+                 ];
+                 
+let enumsMap: {[index: string]: any} = {
+}
+
+let typeMap: {[index: string]: any} = {
+    'APIError': APIError,
+    'CompositeChildModel': CompositeChildModel,
+    'CompositeEntityModel': CompositeEntityModel,
+    'EntityModel': EntityModel,
+    'IntentModel': IntentModel,
+    'LuisResult': LuisResult,
+    'Sentiment': Sentiment,
+}
+
+export class ObjectSerializer {
```

Comment:
We need tests for this class as well; it looks like it's [exposed](https://github.com/microsoft/botbuilder-js/pull/1107/files#diff-583a03304a5e52260ab6e44f50ecd0c5R1-R14).

Reply:
Now it's not, it shouldn't have been exposed